evaluation/round_split/scripts/sort-sort.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Is it possible that you use the scripts and inputs from benchmarks instead of copying them again under round_split?